source/extensions/filters/listener/tls_inspector/tls_inspector.cc

```diff
@@ -228,14 +228,20 @@ std::vector<absl::string_view> Filter::getAlpnProtocols(const unsigned char* dat
     return protocols;
   }
 
-  absl::string_view str(reinterpret_cast<const char*>(data));
   for (unsigned int i = 0; i < len;) {
```

Comment:
should we put a cap on the value of len that we are willing to accept?

Reply:
At the point where getAlpnProtocols() is called the buffer has already been allocated and populated off the wire, so why would we reject what's in it?

Reply:
makes sense resolving